README.md

```diff
@@ -1,23 +1,25 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Bookstore ---- Reducers and Action
+# Bookstore: Use Redux in React components.
```

Comment:
Great job @chuaindia 🤝

- [ ] [Optional] You may kindly fix your MIT license link to make your readme file more professional. At the moment the link appears broken.

Reply:
Ok. I will do that.